.gitignore

```diff
@@ -0,0 +1 @@
+*.env
```

Comment:
This means that all files with this extension will be in gitignore. In my case * not needed.